lib/Extension/ClassMover/Rpc/ReferencesHandler.php

```diff
@@ -268,19 +268,46 @@ private function performFindOrReplaceReferences(
     ) {
         switch ($symbolContext->symbol()->symbolType()) {
             case Symbol::CLASS_:
-                return $this->classReferences($filesystem, $symbolContext, $source, $replacement);
+                [$source, $references] = $this->classReferences($filesystem, $symbolContext, $source, $replacement);
+                break;
             case Symbol::METHOD:
-                return $this->memberReferences($filesystem, $symbolContext, ClassMemberQuery::TYPE_METHOD, $source, $replacement);
+                [$source, $references] = $this->memberReferences($filesystem, $symbolContext, ClassMemberQuery::TYPE_METHOD, $source, $replacement);
+                break;
             case Symbol::PROPERTY:
-                return $this->memberReferences($filesystem, $symbolContext, ClassMemberQuery::TYPE_PROPERTY, $source, $replacement);
+                [$source, $references] = $this->memberReferences($filesystem, $symbolContext, ClassMemberQuery::TYPE_PROPERTY, $source, $replacement);
+                break;
             case Symbol::CONSTANT:
-                return $this->memberReferences($filesystem, $symbolContext, ClassMemberQuery::TYPE_CONSTANT, $source, $replacement);
```

Comment:
could rename this method to f.e. `doPerformFindOrReplace` and decorate it with the sorting functionality.

Reply:
Done, if I understood what you meant ;)